### PR TITLE
Pull request for WAZO-1698-conference-user-events

### PR DESCRIPTION
**Pull request by sduthil**
conferences: add talking/stop event for users
conferences: add missing headers on participant left event

### CODE REVIEW

**Comment by sduthil** [comment]
Depends-On: https://github.com/wazo-platform/xivo-bus/pull/19

**Comment by fredericlepied** [comment]
recheck

File: wazo_calld/plugins/conferences/bus_consume.py [hunk 1]
```diff
@@ -104,6 +105,7 @@ def _notify_participant_unmuted(self, event):
             'language': event['Language'],
             'call_id': event['Uniqueid'],
         }
+        raw_participant['user_uuid'] = event.get('ChanVariable', {}).get('XIVO_USERUUID')
```
Comment (fblackburn1) [inline wazo_calld/plugins/conferences/bus_consume.py:108]:
```suggestion
            'call_id': event['Uniqueid'],
            'user_uuid': event.get('ChanVariable', {}).get('XIVO_USERUUID'),
        }
```
And for all other same changes

Comment (sduthil) [reply wazo_calld/plugins/conferences/bus_consume.py]: Fixed, thank you

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Pull request merged**